### PR TITLE
added description and state prop 

### PR DESCRIPTION
**Pull request by AlecVanSteenbrugge**
Autocomplete can pass the description and state prop so users will be notified when they have a faulty result in there TextField.

### CODE REVIEW

File: packages/antwerp-ui/react-components/src/lib/molecules/autocomplete/Autocomplete.types.ts [hunk 1]
```diff
@@ -8,6 +8,8 @@ export interface AutocompleteProps {
   value?: string | string[];
   noResultsText?: string;
   qa?: string;
+  description?:string;
+  state?:'error'|'success'
```
Comment (TriangleJuice) [inline packages/antwerp-ui/react-components/src/lib/molecules/autocomplete/Autocomplete.types.ts]:
```suggestion
  description?: string;
  state?: 'error' | 'success'
```

**Review by TriangleJuice — CHANGES_REQUESTED** [review]
Thanks for your pull request and helping us creating the most solid design system for the city of Antwerp! I've suggested a few changes, related to linting rules.

Could you please also do the following things:

- Add the new feature to the documentation (in Autocomplete.stories.jsx)
- Add a CHANGELOG entry (under "Unreleased")

**Review by TriangleJuice — APPROVED** [review]

**Comment by NicolasYapla** [comment]
@AlecVanSteenbrugge Can you update this in the documentation of Autocomplete as well? It's in the Autocomplete.stories.jsx file.

**Review by TriangleJuice — CHANGES_REQUESTED** [review]
Er zijn nog git merge issues

**Comment by AlecVanSteenbrugge** [comment]
> Er zijn nog git merge issues

i've corrected this in 37ceff9, my appologies but now there shoudn't be any merge issues.

**Review by TriangleJuice — APPROVED** [review]

**Comment by TriangleJuice** [comment]
> > Er zijn nog git merge issues
> 
> i've corrected this in [37ceff9](https://github.com/a-ui/core_components_react/commit/37ceff9f8c1f4b2ea6b431329ffb54536534f2a8), my appologies but now there shoudn't be any merge issues.

No apologies needed! Thank you for fixing it.
@NicolasYapla Ok for you now?

**Comment by NicolasYapla** [comment]
Looks good, I merge it. Will be published in next version.

**Pull request merged**